app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,61 @@
+class CustomersController < ApplicationController
+  layout 'settings'
+  before_action :set_customer, only: [:edit, :update, :destroy]
+  before_action :require_organization
+
+  def new
+    @customer = current_organization.customers.build
+  end
+
+  def edit
+  end
+
+  def index
+    @customers = current_organization.customers
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @customers , only: [:id, :name] }
+    end
+  end
+
+  def create
+    @customer = current_organization.customers.build(customer_params)
+
+    if @customer.save
+      redirect_to customers_path, notice: 'Customer was successfully created.'
+    else
+      render action: 'new'
+    end
+  end
+
+  def autocomplete
+    @customers = current_organization.customers.pluck(:id, :name)
```

Comment:
@shir What is supposed to be in `term`? What kind of conditions? Something like
{name: customer_name}, so that we could 
``` @customers = current_organization.customers.where(name: customer_name) ? But will return only 1 customer